src/utils/handlePlaylists.js

```diff
@@ -0,0 +1,100 @@
+import axios from "axios";
+import checkLogin from "./checkLogin";
+import toast from "./toast";
+
+const addAPlaylist = async (playlistData, tokenName, isLoggedIn, dispatchUser) => {
+    if (checkLogin(isLoggedIn)) {
+      try {
+        const res = await axios.post(
+          "/api/user/playlists",
+          { playlist: playlistData },
+          {
+            headers: {
+              authorization: localStorage.getItem(tokenName),
+            },
+          }
+        );
+        if (res.status === 200 || res.status === 201) {
+          const { playlists } = res.data;
+          dispatchUser({type: "UPDATE_PLAYLIST", payload: playlists})
+          toast({type:"success", message: "Playlist Created"})
+        }
+      } catch (err) {
+        console.log("Something bad happened", err);
+        toast({type:"error", message: "Couldn't complete the reuqest"})
```

Comment:
Typo here

Reply:
Thanks for pointing this out :D
Will amend it !